From 1ceea3b181c04380d4fcc119b954134b401e79af Mon Sep 17 00:00:00 2001
From: allen yan <yanwei@marvell.com>
Date: Mon, 26 Dec 2016 16:49:20 +0800
Subject: [PATCH 0787/1345] dts: a3700: add uart1 mpp definition

commit  a93d833c5e5986af1e590b824ade6ab31304c9ce from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- Add UART1 mpp settings (North bridge gpio18 and gpio19) in mpp
  dt node. e.g. mpp1[18] for UART1 TX and mpp1[19] for UART1 RX.

Change-Id: Id64ffeceed86b695a4fb807027fccaeee89713be
Signed-off-by: allen yan <yanwei@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/34991
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hua Jing <jinghua@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 3276ceb..09ce177 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -311,6 +311,11 @@
 							"mpp31", "mpp32", "mpp33", "mpp34", "mpp35";
 					marvell,function = "mmc";
 				};
+
+				uart1_pins: uart1_pins {
+					marvell,pins = "mpp18", "mpp19";
+					marvell,function = "uart2";
+				};
 			};
 
 			sgateclk: south-bridge-gate-clk@18014 {
-- 
1.7.9.5

